docs/changelog.rst

```diff
@@ -3,6 +3,45 @@ Changelog
 
 .. scriv-insert-here
 
+.. _changelog-1.0.11:
+
+funcx & funcx-endpoint v1.0.11
+------------------------------
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Created `FuncxWebClient` and `FuncXClient` methods to delete endpoints
+  from the web service.
+- Added a `--force` flag for the `funcx-endpoint delete` command, which
+  ensures that the endpoint is deleted locally even if the web service
+  returns an error or is unreachable.
+
```

Comment:
This is reST, so these should be double backticks, not singular:

```diff
-- Created `FuncxWebClient` and `FuncXClient` methods to delete endpoints
+- Created ``FuncxWebClient`` and ``FuncXClient`` methods to delete endpoints
```

There are a few of these in this changelog ...

Reply:
Fixed.